### PR TITLE
🌱 Update default version to 0.13.0

### PR DESCRIPTION
**Pull request by qiujian16**
[comment]: # ( Copyright Contributors to the Open Cluster Management project )

## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/helpers/version/version.go [hunk 1]
```diff
@@ -44,66 +44,6 @@ func GetVersionBundle(version string) (VersionBundle, error) {
 
 	// predefined bundle version
 	// TODO: automated version tracking
-	versionBundleList["0.5.0"] = VersionBundle{
```
Comment (zhujian7) [inline pkg/helpers/version/version.go:47]: so this means we only support n-3 versions?

File: pkg/helpers/version/version.go [hunk 2]
```diff
@@ -136,6 +76,17 @@ func GetVersionBundle(version string) (VersionBundle, error) {
...
 	}
 
+	versionBundleList["0.13.0"] = VersionBundle{
+		Registration:             "v0.13.0",
+		Placement:                "v0.13.0",
+		Work:                     "v0.13.0",
+		Operator:                 "v0.13.0",
+		AddonManager:             "v0.13.0",
+		AppAddon:                 "v0.13.0",
+		PolicyAddon:              "v0.13.0",
```
Comment (gparvin) [inline pkg/helpers/version/version.go:86]: I don't see updated versions for any of the components (not the ones I checked).  Is the 0.13.0 release in the works now?

File: pkg/helpers/version/version.go (hunk 1 above)
Comment (qiujian16) [reply pkg/helpers/version/version.go:47]: I think so, we can not support earlier version since the change on operator manifests.

File: pkg/helpers/version/version.go (hunk 2 above)
Comment (qiujian16) [reply pkg/helpers/version/version.go:86]: it is in the progressing Operator/AddonManager have 0.13.0 already.

File: VERSION.txt [hunk 3]
```diff
@@ -1 +1 @@
-v0.7.2
+v0.8.0
```
Comment (zhujian7) [inline VERSION.txt:1]: Is this file necessary? can we remove it?

Comment (qiujian16) [reply VERSION.txt:1]: let's add a todo...

Comment (dhaiducek) [reply VERSION.txt:1]: It's referenced here for CLI versioning:
https://github.com/open-cluster-management-io/clusteradm/blob/bb97f150a2340e6cfe7d407c7a71af8c39eaeaf3/version.go#L9
